### DISCUSSION TITLE
using WID with a reed?

### DISCUSSION THREAD

**Discussion by JeffBurris** [edwardkort/WWIDesigner#152] [Ideas]
I use WID for NAF and absolutely rely on it. Super appreciative, including Edward Kort's personal help, a couple of years ago.

I was wondering if WID could be reasonably used for one of those compact "saxophones" that are like a wooden or bamboo flute but they have a sax mouthpiece strapped on.

How close might it be if I replaced splitting edge position with the vibrating end of the reed, etc?

Any thoughts?
Thanks!

**Comment by edwardkort** [comment]
Hi Jeff,
A reed instrument has different resonance characteristics than a NAF due to its closed-end mouthpiece. It is typically half the bore length relative to a flute of the same pitch. That said, WIDesigner has a Reed Study Model to aid in the design of such instruments.

In the Options/Preferences dialog of the program, select Reed Study. You will then see a different set of optimizers than you have used in the NAF Study. The documentation (written by Burton Patkau, as well as the code) for using the Reed Study may be found in our wiki: https://github.com/edwardkort/WWIDesigner/wiki/Optimizers-in-the-Reed-Study-Model

Let us know if you have any questions after reading and trying the above.

Cheers,
Edward Kort

**Reply by JeffBurris** [reply]
Oh my goodness I can't believe I missed or forgot WID had a reed study altogether, thanks. 

I'm having to concentrate on getting NAF's out there more efficiently, but this is very exciting for when I have time in the spring. WID is such a great point of departure for me for NAF's... I get it as close to +/- 5% as I can for each note then, in reality, I drill small to hand-tune up, sometimes nudging the hole in one direction a little (usually hole 3, when doing the 2 groups of holes) and they come out greatly, despite a straight bore and 2 groups of equally-spaced holes don't love to yield XXX OOO and XXX OXO both in tune at the same time. Fipple Factor helps so much, overall.
It's great to hear from you, Mr. Kort. 
Thanks,
Jeff Burris

**Comment by bhp1** [comment]
> I was wondering if WID could be reasonably used for one of those compact "saxophones" that are like a wooden or bamboo flute but they have a sax mouthpiece strapped on.

The reed study model, as Edward has pointed out, should definitely work on such an instrument.  But ...
 
> How close might it be if I replaced splitting edge position with the vibrating end of the reed, etc?

I'm still not sure where to mark the "mouthpiece position" on a sax or clarinet style single-reed mouthpiece.  See [Mouthpiece Geometry](https://github.com/edwardkort/WWIDesigner/wiki/Working-with-the-Reed-Study-Model#mouthpiece-geometry) for considerations.  Perhaps you could try a couple of possibilities and see which gives the best predictions: the vibrating tip of the reed, or the uppermost clamp at the start of the non-vibrating part of the reed.  My inclination is to mark it at the non-vibrating part of the reed rather than the tip.

The reed study model is still a work in progress, so let us know how it works.

**Reply by JeffBurris** [reply]
Thanks so much to you, as well, bhp1... when I can get to this exciting project I'll record all my real-life results (and keep them with the corresponding software files for comparison) in case they can help in any way. 

I now see the other posts about reeds, here, too. Great to go over in conjunction with the tutorials when I try this.
